dali/util/image.h

```diff
@@ -83,7 +83,7 @@ int outCHWImage(const vector<T> &tmp, int h, int w, int c,
   return static_cast<int>(tmp[k*h*w + i*w + j]*scale + bias);
 }
 
-typedef int (*outFunc)(const vector<double> &tmp, int h, int w, int c,
+typedef int (*outFunc)(const vector<uint8_t> &tmp, int h, int w, int c,
```

Comment:
`using outFunc = std::function<...>;`

Reply:
I thought we didnt want to use std::function when typedef is enough it because of some ABI incompat issues

Reply:
alright, lets leave it